lib/Config.ts

```diff
@@ -61,6 +63,10 @@ class Config {
     this.rpc = {
       port: 8886,
     };
+    this.api = {
```

Comment:
* I think we should put `api` inside of `rpc` property, because it's not a separate module
* Perhaps should call it `webProxy`. `api` is too generic
* instead of `listen`, lets use `disable`, which is our usual pattern